docs/Reply.md

```diff
@@ -109,6 +110,14 @@ Invokes the custom not found handler.
 reply.callNotFound()
 ```
 
+<a name="getResponseTime"></a>
+### .getResponseTime()
+Invokes the custom response time getter to calculate the amount of time passed since a reply has been sent.
```

Comment:
Shouldn’t this be “since the request was started”?

Reply:
I don't _think_ so, the timer actually currently gets setup in `setupResponseListeners`, and I didn't want to modify this behaviour. see: https://github.com/fastify/fastify/pull/1697/files#diff-d2fdf63c1d7f1d9f68270b6dfa8c930cL430

Reply:
However, I'm betting you understand the internals better and can change wording if you want :)

Reply:
It’s meant to track the time it has taken to process the request and send the reply. Your is stating that it is a measure of time since the reply was sent. That doesn’t really make any sense because once a reply has been sent there is no longer any context to measure.